style.css

```diff
@@ -1,129 +1,239 @@
-*{
-    margin: 0;
-    padding: 0;
-    
-}
-.header{
-    min-height:  100vh;
-    width: 100%; 
-    background-image: linear-gradient(rgba(4, 9, 30, 0.7),rgba(4, 9, 30, 0.7)),url(Images/banner.png);
-    
-    background-position: center;
-    background-size: cover;
-    position: relative;   
-    
-}
-nav{
-    
-    padding: 2% 3%;  
-    display: flex;   
-    justify-content: space-between;
-    align-items: center;    
-}
-nav img{
-    width: 150px;  
-}
-.nav-links{
-    flex: 1;
-    text-align: right;  
-    font-family: 'Poppins', sans-serif;    
-    
-}
-.nav-links ul li{
-    list-style: none;
-    display: inline-block;
-    padding: 8px 12px;
-    position: relative;   
+* {
+  margin: 0;
+  padding: 0;
+  font-family: BlinkMacSystemFont, -apple-system, Segoe UI, Roboto, Helvetica, Arial, sans-serif;
 }
 
-.nav-links ul li a{
-    color: #fff;
-    text-decoration: none;
-    font-size: 14px;
+.header {
+  min-height: 100vh;
+  width: 100%;
+  background-image: linear-gradient(rgba(4, 9, 30, 0.7), rgba(4, 9, 30, 0.7)), url(images/banner.png);
```

Comment:
I'd suggest to test how it displays in different browsers and if you want to support most popular ones (mozilla, chrome, safari, opera) make sure to add vendor prefixes. some context:

1) https://developer.mozilla.org/en-US/docs/Glossary/Vendor_Prefix
2) https://stackoverflow.com/questions/28184562/make-my-background-image-supported-by-all-browsers/28189270